pubspec.yaml

```diff
@@ -1,16 +1,17 @@
 name: dashbook
 description: A showcase/sandbox library to help you build isolated widgets in Flutter
-version: 0.0.11
+version: 1.0.0
```

Comment:
**thought:** this should be fine but we may end up going to `0.1.0` when the library bumps.